control-plane/pkg/reconciler/testing/objects_broker.go

```diff
@@ -194,6 +194,16 @@ func WithBootstrapServerStatusAnnotation(servers string) reconcilertesting.Broke
 	}
 }
 
+func WithSecretStatusAnnotation(name string, namespace string) reconcilertesting.BrokerOption {
+	return func(broker *eventing.Broker) {
+		if broker.Status.Annotations == nil {
+			broker.Status.Annotations = make(map[string]string, 10)
+		}
+		broker.Status.Annotations[security.AuthSecretNameKey] = name
+		broker.Status.Annotations[security.AuthSecretNamespaceKey] = namespace
+	}
+}
```

Comment:
For the broker, I see triggerv2.go uses (correctly) `spec.config.namespace`, so we can reduce the function to just:
```suggestion
func WithSecretStatusAnnotation(name string) reconcilertesting.BrokerOption {
	return func(broker *eventing.Broker) {
		if broker.Status.Annotations == nil {
			broker.Status.Annotations = make(map[string]string, 10)
		}
		broker.Status.Annotations[security.AuthSecretNameKey] = name
	}
}
```